Makefile

```diff
@@ -30,32 +33,42 @@ DESTPATH    := $(DESTDIR)$(PREFIX)
 BINPATH     := $(DESTDIR)$(EXEC_PREFIX)/bin
 SHAREPATH   := $(DESTPATH)/share
 MANDIR      := man/man1
-GCB         := $$(git rev-parse --abbrev-ref HEAD)
-NOMASTER    := $$([[ $(GCB) != master ]] && echo -$(GCB))
-DISTNAME    := compiled
-DISTDIR     := dist
+DISTNAME    := $(PROG)-$$(cat $(VERFILE))-$(SYSTEM)
+TMPDIR      := compiled
```

Comment:
Since this is a name out of the compiled output, TMPDIR is misleading here.
Rename this to COMPILEDIR.

Add comment that this gets overriden by dist*